CafeApp/screens/productsScreen.tsx

```diff
@@ -1,33 +1,56 @@
 import {NativeStackScreenProps} from '@react-navigation/native-stack';
 import React from 'react';
-import {View} from 'react-native';
+import {FlatList, SafeAreaView, StyleSheet, View} from 'react-native';
 import {RootStackParamList} from '../App';
-import {Button, ButtonTypes} from '../components/buttonWrapper';
+import {PressableTile} from '../components/pressableTile';
+import {globoProducts} from '../database/productsDB';
+
 type ProductsScreenProps = NativeStackScreenProps<
   RootStackParamList,
   'Products'
 >;
+
+interface ProductProps {
```

Comment:
interfaces with word props is usually being used for jsx components. This looks to me like Product entity, so maybe it is worth extracting it to separate file and put it to "models" file?